include/asterisk/app.h

```diff
@@ -1196,9 +1196,54 @@ struct ast_group_info;
 /*! \brief Split a group string into group and category, returning a default category if none is provided. */
 int ast_app_group_split_group(const char *data, char *group, int group_max, char *category, int category_max);
 
+/* Create group meta if it doesn't exist  */
+int ast_app_group_add_meta_ifneeded(const char *group, const char *category);
+
+/* This removes the the group metadata for a group@category if there are no more channels in the group@category */
```

Comment:
doxygen + rephrase to imperative ("Remove the group metadata...")

Reply:
Fixed